testsuite/gnat2goto/tests/arrays_multidimensional_type_instantiation_failure/test.opt

```diff
@@ -0,0 +1 @@
+ALL XFAIL CBMC fails with invariant failure "Condition: op.operands()[number].type() == expr.type()"
```

Comment:
missing newline

Reply:
I thought I'd recently set emacs to save with newline? Grrrrrr...